node/node-4-tests.ts

```diff
@@ -249,18 +249,24 @@ function bufferTests() {
 /// Url tests : http://nodejs.org/api/url.html
 ////////////////////////////////////////////////////
 
-url.format(url.parse('http://www.example.com/xyz'));
-
-// https://google.com/search?q=you're%20a%20lizard%2C%20gary
-url.format({
-    protocol: 'https',
-    host: "google.com",
-    pathname: 'search',
-    query: { q: "you're a lizard, gary" }
-});
+namespace url_tests {
+    {
```

Comment:
@zhengbli It just splits into each test. It's because I follows the test style in this test definition.  If you had review other module tests of namespace, you will find they have similar style with each other.

If you consider the block may be removed, it's okay. But my opinion is when new contributor will code new test in module tests, the block can isolate each test and get protected. I think it's good test style in this test definition, so I follow it. :smile: